src/acquisition/covidcast/csv_importer.py

```diff
@@ -118,13 +120,13 @@ def find_csv_files(scan_dir, issue=(date.today(), epi.Week.fromdate(date.today()
         # safe to ignore this file
         continue
 
-      print('file:', path)
+      logger.info(event='covidcast-format CSV file path :', file=path)
```

Comment:
i'm still confused about what this log statement is for. when we were using `print`, it was important to print the file path before starting the next section, so you could tell from reading stdout what file any subsequent prints referred to. now that we're including the file= argument in the structured log, that isn't really necessary. is this log statement serving some other purpose, or can we just get rid of it?

Reply:
I think this statement is like a double check before moving to next part after check the file path is ended with ".csv". Can I delete this statement?

Reply:
yeah, go ahead and delete it; it's not needed